Private/New-Dictionary.ps1

```diff
@@ -101,12 +101,39 @@ function New-Dictionary {
         [VulnerableConfigurationItem]@{
             Name = 'ESC8'
             Category = 'Escalation Path'
-            Subcategory = 'NTLM Relay to AD CS HTTP Endpoints'
+            Subcategory = 'AD CS HTTP Endpoints vulnerable to NTLM Relay'
```

Comment:
Hate me, but that lowercase 'V' doesn't follow the consistent use of title case above. :)

Reply:
Done!